timedate/src/main/java/com/example/timedate/TimeDate.java

```diff
@@ -4,12 +4,22 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 @Controller
 public class TimeDate {
 
     @GetMapping("/hello")
     public String hello(Model model) {
-        model.addAttribute("tag", "Hello, World!");
+
+        LocalDateTime nowDate = LocalDateTime.now();
+        DateTimeFormatter dtf2 = DateTimeFormatter.ofPattern("yyyy年MM月dd日 HH時mm分ss秒 E曜日");
+        String formatNowDate = dtf2.format(nowDate);
+
+        model.addAttribute("tag", "TimeDate, World!");
```

Comment:
固定の文字列を返却するならば画面側に文字列を表示するようにしてしまう方が良いかなと思います！
もしJava側からで文字列を渡したいなら、変数に文字列を入れて、その変数を画面側に返すようにすると変更しやすいプログラムになるように思います！

```suggestion
      String tag = "TimeDate World!";

        model.addAttribute("tag", tag);
```